hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -649,12 +649,25 @@
 
     // CLKMGR tests:
     {
-      name: chip_sw_clk_off_trans
+      name: chip_sw_clk_idle_trans
       desc: '''Verify the ability to turn off the transactional clock via SW.
 
-             Ensure that activity in any of the IPs running on this clock prevents the clock from
-             actually being turned off. Verify that turning off this clock does not affect the other
-             derived clocks.
+            Ensure that activity in any of the IPs running on this clock prevents the clock from
+            actually being turned off until the activity is completed, at which point the clock is
+            turned off.  Verify it is off via spinwait in hints_status CSR, and then check the state
+            of the transactional unit to correspond to a completed activity.  Verify that turning
+            off this clock does not affect the other derived clocks.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sw_clk_off_trans
+      desc: '''Verify the turned off transactional units.
+
+            Verify CSR accesses do not complete in units that are off.  Using the watchdog timers,
+            turn off a transactional unit's clock, issue a CSR access to that unit, verify a watchdog
+            event results, and verify the rstmgr crash dump info records the CSR address.
```

Comment:
awesome!

Reply:
we should also double check the recorded PC (somehow) to see if it's useful data.

Reply:
Would you please add that as a `TODO(#bug):` item here in the testpoint description and have an issue filed so that we think about this?

Reply:
Added it as a "stretch" goal.